loggers/printlogger/printLogger.go

```diff
@@ -0,0 +1,49 @@
+package printlogger
+
+import (
+	"encoding/json"
+	"fmt"
+
+	logger "github.com/Clever/wag/loggers/waglogger"
+)
+
+//NewLogger creates a logger for id that produces logs at and below the indicated level.
+//Level indicated the level at and below which logs are created.
+func NewLogger(id string, level string) logger.WagClientLogger {
```

Comment:
this can return PrintlnLogger and then this package has no dependencies

Reply:
Oooh, nice catch. Thanks!